scripts/game.js

```diff
@@ -0,0 +1,64 @@
+var rows = 50;
+var cols = 100;
+
+var playing = false;
+
+// define grid using 'new' operator to create an instance of the grid object
+var grid = new Array(rows);
+var nextGrid = new Array(rows);
+
+var timer;
+var reproductionTime = 100;
+
+
+
+function buildGrid() {
+  for (var i = 0; i < rows; i++) {
+    grid[i] = new Array(cols);
```

Comment:
a `.forEach()` method might be helpful here! 
Since you're not using the `i` value beyond identifying the index `forEach` helps with readability & speed
ala:
`grid.forEach(row => { row = new Array(cols) })`